src/views/product-landing/server.ts

```diff
@@ -1,100 +1,100 @@
-  // TODO: at build time from said URL, to ensure the displayed
-  // TODO: content is accurate, while not needing to be
-  // TODO: manually kept up to date?
-  // Asana task: https://app.asana.com/0/1201010428539925/1201646299837754/f
-  const usedHeadings = []
-  CONTENT.blocks = CONTENT.blocks.map((block) => {
-    switch (block.type) {
-      case 'heading':
-        // augment heading blocks with a consistent slug,
-        // used for anchor linking
-        // if the resulting slug is not unique, append an
-        // integer suffix to make sure that it is
-        // eslint-disable-next-line no-case-declarations
-        const baseSlug = slugify(block.heading, { lower: true })
-        // eslint-disable-next-line no-case-declarations
-        let slug = baseSlug
-        // eslint-disable-next-line no-case-declarations
-        let slugMakeUniquePrefix = 0
-        while (usedHeadings.indexOf(slug) !== -1) {
-          slugMakeUniquePrefix++
-          slug = `${baseSlug}-${slugMakeUniquePrefix}`
-        }
-        usedHeadings.push(slug)
-        return { ...block, slug }
-      case 'cards':
-        // TODO: instead of the below, should likely use product context
-        // TODO: in IconTile for default brandColor (rather than current
-        // TODO: behavior of static default to "neutral"). Would eliminate
-        // TODO: the need to add iconBrandColor to cards (unless doing so
-        // TODO: as an explicit override).
-        // ensure cards with icons have an iconBrandColor,
-        // falling back to the current product if not set
-        /* eslint-disable-next-line no-case-declarations */
-        let defaultIconColor
-        if (product.slug === 'hcp' || product.slug === 'sentinel') {
-          defaultIconColor = 'neutral'
-        } else {
-          defaultIconColor = product.slug
-        }
-        return {
-          ...block,
-          cards: block.cards.map((card) => ({
-            ...card,
-            iconBrandColor: card.iconBrandColor || defaultIconColor,
-          })),
-        }
-      default:
-        return block
-    }
-  })
-
-  const navData = [
-    ...product.sidebar.landingPageNavData,
-    { divider: true },
-    ...product.sidebar.resourcesNavData,
+  /**
+   * TODO: content is raw placeholder for now
+   * - Need to add "headings" that aren't placeholders.
+   *   These will be derived from content blocks
+   *   (mainly heading blocks, but also others, based on design intent)
+   * - Need to augment things like tutorialSlugs to fill in that data
+   *   Will close: https://app.asana.com/0/1201010428539925/1201654639085764/f
+   * - Need to build out the client-side components, whose props APIs will
+   *   inform what content structure we'll want to return for use on the client.
+   *   Intent being to return the minimum data necessary to render
+   *   all content block components on the page.
+   */
+  const placeholderHeadings = [
```

Comment:
I've stripped back the server function to the minimum needed to stub things out, in preparation for rebuilding the page with the revised designs.

These placeholder headings will look ok, but won't function correctly. Intent here is to hook up the sidecar table-of-contents in a subsequent PR (likely re-using utilities from the `Product Tutorials Landing` view).